recipes/tiledb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: tiledb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB/archive/{{ version }}.tar.gz
+  sha256: 6e613d222c5f15537dbbd0cf121138c27d480780cc56acd5a7d1ff9f9ce0fe56
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.2
+    - zlib 1.2.11
+    - bzip2
+    - zstd
+    - lz4-c 
+    - blosc 1.12.0
+  run:
+    - zlib 1.2.11
+    - bzip2
+    - zstd
+    - lz4-c
+    - blosc 1.12.0
+
+test:
+  commands:
+    - test -e $PREFIX/include/tiledb/tiledb.h  # [unix]
+    - test -e $PREFIX/include/tiledb/tiledb  # [unix]
+    - test -e $PREFIX/lib/libtiledb.a  # [unix]
+    - test -e $PREFIX/lib/libtiledb.so  # [linux]
+    - test -e $PREFIX/lib/libtiledb.dylib  # [osx]
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+about:
+  home: http://tiledb.io
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'TileDB sparse and dense multi-dimensional array data management'
+  description: |
+    TileDB is an efficient multi-dimensional array management system which introduces 
+    a novel on-disk format that can effectively store dense and sparse array data with 
+    support for fast updates and reads. It features excellent compression, and an efficient 
+    parallel I/O system with high scalability.
+  doc_url: http://docs.tiledb.io
+  dev_url: https://github.com/TileDB-Inc/TileDB
+
+extra:
+  recipe-maintainers:
+    - stavrospapadopoulos
+    - jakebolewski
+    - tdenniston
```

Comment:
@tdenniston, are you happy to [be a co-maintainer for the conda-forge `tiledb` recipe](https://conda-forge.org/docs/recipe.html#maintainer-role)?

Reply:
Yes.